src/ir/irutil/irtutil.go

```diff
@@ -66,6 +66,29 @@ func FmtNode(n ir.Node) string {
 	return irfmt.Node(n)
 }
 
+// Find searches for a node in the passed subtree.
+func Find(what ir.Node, where ir.Node) bool {
```

Comment:
Is this functions used?

Reply:
No, left for the future.

Reply:
By the way, we have already discussed that it is possible 
to collect information about variables and check them here, 
but I don’t remember exactly, but there were some problems 
that I could not solve.

Reply:
That's not a good reason to keep unused code. :)

Reply:
These util functions are almost never removed, even if unused. It's a good way to keep sanity with not adding more if you don't need them that much.

Reply:
Removed.